### PR TITLE
[docs] Fix link for SshKeyPair in README

### PR DESCRIPTION
**Pull request by mansikulkarni96**
Updated Readme to point the correct link for updating shheyPair,
now mentioned in create_test.go file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ export AWS_SHARED_CREDENTIALS_FILE=<path to aws credentials file>
 export CLUSTER_ADDR=<cluster_name, eg: ravig211.devcluster.openshift.com>
 export KUBE_SSH_KEY_PATH=<path to ssh key>
 ```
-- Update the keypair mentioned at [sshKeyPair](https://github.com/openshift/windows-machine-config-operator/blob/42593c5d2eb798b572c58b5debafc4c392d1f967/test/e2e/wmco_test.go#L59) to match with what 
+- Update the sshKeyPair mentioned in [create_test.go](https://github.com/openshift/windows-machine-config-operator/blob/master/test/e2e/create_test.go) to match with what
```
Comment (PratikMahajan) [inline README.md:24]: you can point to the line number where this needs to be updated

Comment (aravindhp) [reply README.md:24]: I suggest we don't do that. Every time the code is moved around we have to remember to update the README.

Comment (mansikulkarni96) [reply README.md:24]: @PratikMahajan refrained from doing that for the same reason.

**Pull request merged**